src/Bulk/ResponseSet.php

```diff
@@ -19,12 +20,9 @@ class ResponseSet extends BaseResponse implements \Iterator, \Countable
     /**
      * @param Response[] $bulkResponses
      */
-    public function __construct(BaseResponse $response, array $bulkResponses)
+    public function __construct(Elasticsearch $response, array $bulkResponses)
     {
-        parent::__construct($response->getData(), $response->getStatus());
-
-        $this->setQueryTime($response->getQueryTime());
```

Comment:
I guess this is not available anymore?

Reply:
Yes, i have to remove property $_queryTime from Response object.